src/gui/dialogs/statistics_dialog.cpp

```diff
@@ -135,32 +156,149 @@ void statistics_dialog::add_damage_row(
 
 	const long long shifted = ((expected * 20) + shift) / (2 * shift);
 	std::ostringstream str;
-	str << damage << " / "
-		<< static_cast<double>(shifted) * 0.1
-		<< "    " // TODO: should probably make this two columns
-		<< (((dsa < 0) ^ (expected < 0)) ? "" : "+")
-		<< (expected == 0 ? 0 : 100 * dsa / expected) << '%';
-
+	str << damage << " / " << static_cast<double>(shifted) * 0.1;
 	item["label"] = str.str();
 	data.emplace("damage_overall", item);
 
 	str.str("");
+	str << (((dsa < 0) ^ (expected < 0)) ? "" : "+")
+		<< (expected == 0 ? 0 : 100 * dsa / expected) << '%';
+	item["label"] = str.str() + spacer;
+	data.emplace("overall_percent", item);
 
 	if(show_this_turn) {
 		const long long turn_shifted = ((turn_expected * 20) + shift) / (2 * shift);
-		str << turn_damage << " / "
-			<< static_cast<double>(turn_shifted) * 0.1
-			<< "    " // TODO: should probably make this two columns
-			<< (((dst < 0) ^ (turn_expected < 0)) ? "" : "+")
-			<< (turn_expected == 0 ? 0 : 100 * dst / turn_expected) << '%';
-
+		str.str("");
+		str << turn_damage << " / " << static_cast<double>(turn_shifted) * 0.1;
 		item["label"] = str.str();
 		data.emplace("damage_this_turn", item);
+
+		str.str("");
+		str << (((dst < 0) ^ (turn_expected < 0)) ? "" : "+")
+			<< (turn_expected == 0 ? 0 : 100 * dst / turn_expected) << '%';
+		item["label"] = str.str() + spacer;
+		data.emplace("this_turn_percent", item);
 	}
 
 	damage_list.add_row(data);
 }
 
+// Return the strings to use in the "Hits" table, showing actual and expected number of hits.
+static std::pair<std::string, std::string> tally(const std::map<int, struct statistics::stats::by_cth_t>& by_cth)
+{
+	unsigned int overall_hits = 0;
+	double expected_hits = 0;
+	unsigned int overall_strikes = 0;
+
+	for(const auto& i : by_cth) {
+		int cth = i.first;
+		overall_hits += i.second.hits;
+		expected_hits += (cth * 0.01) * i.second.strikes;
+		overall_strikes += i.second.strikes;
+	}
+
+	std::ostringstream str, str2;
+	str << overall_hits << " / " << expected_hits;
```

Comment:
@Pentarctagon https://github.com/wesnoth/wesnoth/issues/4066#issuecomment-491360346

> Otherwise, I'd say to use some other separator than `/`. I can imagine there being a fair number of people confused about how 2 divided by 2.7 somehow ends up at 39%. Maybe display it as `2 (2.7)` instead? 17/25.2 is at least fairly straight forward as to how they are related to the -32%.

Reply:
Quoting from [the forums](https://forums.wesnoth.org/viewtopic.php?f=12&t=49785&p=642417#p642417):

> > (Mawmoocn) For visual UI, instead of 80 / 83.3, it could be, 80 = 83.3, [80 = 83.3] [28.0%] [72.0%], [80 - 83.3 (28.0%)] [72.0%], [80 | 83.3 (28.0%)]. 

> Thanks. I wouldn't use = here because 80 is not equal to 83.3. I suppose | is an option but I don't think it's very intuitive. Hmm, maybe "80 (-3.3)" or "83.3 ⊨ 80"? :thinking:

Reply:
@Pentarctagon Just to tie up loose ends, I assume this is resolved now that the dialog shows two percentages instead of one?